api/pkg/db/migration/migration_template.go

```diff
@@ -14,18 +14,21 @@
 
 package migration
 
-import (
-	"github.com/go-gormigrate/gormigrate/v2"
-	"github.com/tektoncd/hub/api/gen/log"
-	"gorm.io/gorm"
-)
+// Commenting this template as it is just a blueprint for adding a new migration and
```

Comment:
Should we completely remove this?

Reply:
or you can use `--skip-files`

Reply:
may be remove this.. we have a few migrations for examples now

Reply:
Yes I think removing will be fine